app/views/accounting/search_transaction.html

```diff
@@ -33,12 +34,17 @@
                     <h4 class="control-label ng-binding">{{'label.heading.daterange' | translate}}</h4>
                     <br>
                     <div class="form-group">
-                        <input class="form-control" type="text" datepicker-pop="dd MMMM yyyy" ng-model="date.first" is-open="opened" min="minDate" max="'2020-06-22'" placeholder="{{ 'label.input.fromdate' | translate }}" style="width: 155px;"/>
+                        <input id="fromDate" class="form-control" type="text" datepicker-pop="dd MMMM yyyy" ng-model="date.first" is-open="opened" min="minDate" max="maxDate" placeholder="{{ 'label.input.fromdate' | translate }}" style="width: 155px;"/>
                     </div>
                     <div class="form-group">
-                        <input class="form-control" type="text" datepicker-pop="dd MMMM yyyy" ng-model="date.second" is-open="opened1" min="minDate" max="'2020-06-22'" placeholder="{{ 'label.input.todate' | translate }}" style="width: 155px;"/>
+                        <input id="toDate" class="form-control" type="text" datepicker-pop="dd MMMM yyyy" ng-model="date.second" is-open="opened1" min="minDate" max="maxDate" placeholder="{{ 'label.input.todate' | translate }}" style="width: 155px;"/>
                     </div>
                 </div>
+                <br>
+                    <div class="col-sm-3">
```

Comment:
Instead of   <div class="col-sm-3"> keep the div class as form group. Otherwise alignment will be distorted